content/guides/component-testing/framework-configuration.md

```diff
@@ -39,18 +39,20 @@ Create React App. We can do this easily using the `react-scripts` plugin
 provided by Cypress. Place the following in `cypress/plugins/index.js`, creating
 the relevant directories.
 
-```js
-// cypress/plugins/index.js
+:::cypress-plugin-example
```

Comment:
The directive needs to be updated to support an attribute that allows us to specify the config property that `setupNodeEvents` displays in. Right now, it's hard-coded to `e2e` but some examples will need it to be `component`. We'll also need an attribute to suppress the `setupNodeEvents can be defined in either the e2e or component configuration` comment.

See the `include` directive source for an example of using attributes.

So changing the above line to this:

```
:::cypress-plugin-example {configProp=component noComment=true}
```

should render like this:

```js
const { defineConfig } = require('cypress')
const injectDevServer = require('@cypress/react/plugins/react-scripts')

module.exports = defineConfig({
  component: {
    setupNodeEvents(on, config) {
      injectDevServer(on, config)
      
      return config
    }
  }
})
```

But any existing examples without those attributes should continue to render as-is.

Once that's done, please update all the examples in this file and the content/guides/component-testing/introduction.md file to use the `{configProp=component noComment=true}` attributes.